avocado/utils/process.py

```diff
@@ -1181,6 +1181,11 @@ def system_output(cmd, timeout=None, verbose=True, ignore_status=False,
                  has a sudo configuration such that a password won't be
                  prompted. If that's not the case, the command will
                  straight out fail.
+    :type sudo: bool
```

Comment:
Thanks for adding these missing docstrings!  While you're at it, a couple of hints and requests:
 * If possible, also add the `param sudo` docstring
 * Ideally, as a good practice, this docstring should be in a separate commit.  To check if it's better to have a given piece of code in a separate commit or not, just ask: "does this hunk contributes to what's described on the commit message?"